api/scripts/update-mapping-files.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
```

Comment:
I just temporary copied this over from the old service. I want to store all this urls in redis as well. We can use another lambda function to update the redis store with new url which we will schedule to be called every week or so.